whois-query/src/main/java/net/ripe/db/whois/query/planner/RpslResponseDecorator.java

```diff
@@ -170,6 +177,10 @@ private Iterable<? extends ResponseObject> filterAuth(Query query, final Iterabl
         });
     }
 
+    private static boolean hasCertificates(List<X509CertificateWrapper> certificates) {
```

Comment:
The method name and the check have opposite meanings, maybe call the method `hasNoCertificates` ?